docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,368 @@
+   | **Hybrid Profile**               | Click in the field and select the cluster profile you [created for your hybrid nodes](#create-cluster-profile-for-hybrid-node-pools). Click **Configure**. Review the layers and click **Confirm**.                             |
+   | **Architecture**                 | Select the architecture type for your edge hosts. Either **AMD64** or **ARM64**.                                                                                                                                                |
+   | **NTP Servers (Optional)**       | The Network Time Protocol (NTP) servers to use for the hybrid nodes. For example, `pool.ntp.org`.                                                                                                                               |
+   | **Nodes (edge hosts)**           | Click **Add Edge Hosts**. Select your edge hosts from the table by clicking the checkbox next to the **Machine ID**. Click **Confirm** once done.                                                                               |
+
+7. Once your edge hosts have been selected, click **Configure** next to each edge host to review and configure
+   individual host options.
+
+   | **Field**                | **Description**                                                                                                                                                                                                                                                                                                                  |
+   | ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | **Host Name (Optional)** | Provide a optional name for the edge host that will be displayed in Palette.                                                                                                                                                                                                                                                     |
+   | **NIC Name**             | Select a specific Network Interface Card (NIC) on the edge host from the **drop-down Menu**, or leave it on **Auto**.                                                                                                                                                                                                            |
+   | **VPN server IP**        | Specify the VPN server's IP if the hybrid nodes in the pool use a VPN. If provided, a static route will be configured on edge hosts to route traffic to the Amazon EKS VPC CIDR through the VPN server. If not specified, ensure your hybrid node network routes traffic to the Amazon EKS VPC CIDR through the default gateway. |
```

Comment:
"Specify the VPN server's IP if the hybrid nodes in the pool use a VPN" - this is not quite accurate.

It is only required if using a VPN _and_ the hybrid node's network doesn't automatically route traffic to the EKS VPC CIDR through the VPN server.